server/src/services/blogs/service.js

```diff
@@ -0,0 +1,83 @@
+import mongoose from 'mongoose';
+import Blogs from '../../models/blogs';
+
+const saveBlog = (req, res, next) => {
+  const { title } = req.body;
+  const { content } = req.body;
+  const userId = req.user._id;
+
+  const blog = {
+    title,
+    content,
+    userId,
+  };
+
+  Blogs.create(blog, (err, blogFromDB) => {
+    if (err) {
+      res.status(500).json({ msg: 'Something went wrong' });
+    }
+    req.blog = blogFromDB;
+    next();
+  });
+};
+
+const fetchBlog = (req, res, next) => {
+  const blogID = req.params.id;
+  const validObjectId = mongoose.Types.ObjectId.isValid(blogID);
+
+  if (validObjectId) {
```

Comment:
I guess it is not necessary to check whether the passed blog ID is a mongo Object Id? We can save one if-else here.

Reply:
I tried without validation, findById throws an error if id is not a valid object id and returns 500 status  code.

Reply:
Ohh!! Alright :+1: